pkg/pxc/encryption/keyring.go

```diff
@@ -0,0 +1,157 @@
+package encryption
+
+import (
+	"bytes"
+	"crypto/rand"
+	"crypto/sha256"
+	"encoding/binary"
+	"fmt"
+
+	"github.com/google/uuid"
+)
+
+const (
```

Comment:
I think that here we can create custom type (for example `type KeyRing struct{}`) with all this vars and with function `New(..)` that will set all default values.

Reply:
Not sure if we need it. I can't see any advantages, only more code to write